backend/protos/xyz/block/ftl/v1/language/language.proto

```diff
@@ -0,0 +1,161 @@
+syntax = "proto3";
+
+package xyz.block.ftl.v1.language;
+
+import "xyz/block/ftl/v1/schema/schema.proto";
+import "xyz/block/ftl/v1/ftl.proto";
+
+option go_package = "github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1/language;languagepb";
+option java_multiple_files = true;
+
+enum LogLevel {
+  DEBUG = 0;
+  INFO = 1;
+  WARN = 2;
+  ERROR = 3;
+}
+
+// Log message from the language service.
+message LogMessage {
+  string message = 1;
+  LogLevel level = 2;
+}
+
+message Metadata {
+  message Pair {
+    string key = 1;
+    string value = 2;
+  }
+  // Universal metadata
+  string name = 1;
+
+  // Language metadata contains any metadata specific to a specific language.
+  repeated Pair LanguageMetadata = 2;
```

Comment:
I'm not sure what's going into this or where it's ending up, but we should probably be using [google.protobuf.Any](https://protobuf.dev/programming-guides/proto3/#any) instead of a bag of values.